e2e-tests/cypress/integration/project-application-documents/PageObjects/PO_ProjectAppDocs.js

```diff
@@ -1,13 +1,51 @@
+const accordionId = 'ui-checkbox-accordion__section-switch--';
+const checkboxId = 'ui-checkbox-accordion__checkboxes-section--';
+
+const filterKeys = {
+	'pre-application': {
+		accordionId: accordionId + 'stage-1',
```

Comment:
this could come from the statement ? 

but the statement is more descriptive of what is seen by the user by using "pre-application" etc